ingest/src/server_mobile.rs

```diff
@@ -214,6 +217,51 @@ impl poc_mobile::PocMobile for GrpcServer {
         }))
     }
 
+    async fn submit_threshold_report(
+        &self,
+        request: Request<RadioThresholdReportReqV1>,
+    ) -> GrpcResult<RadioThresholdReportRespV1> {
+        let timestamp = Utc::now().timestamp_millis() as u64;
+        let event = request.into_inner();
+        let hotspot_pubkey = event.hotspot_pubkey.clone();
+        let cbsd_id = event.cbsd_id.clone();
+        let threshold_timestamp = event.threshold_timestamp;
+
+        let report = self
+            .verify_public_key(event.carrier_pub_key.as_ref())
+            .and_then(|public_key| self.verify_network(public_key))
+            .and_then(|public_key| self.verify_signature(public_key, event))
+            .map(|(_, event)| RadioThresholdIngestReportV1 {
+                received_timestamp: timestamp,
+                report: Some(event),
+            })
+            .map_err(|status| {
+                tracing::debug!(
+                    hotspot_pubkey = ?hotspot_pubkey,
+                    cbsd_id = ?cbsd_id,
+                    threshold_timestamp = %threshold_timestamp,
+                    status = %status
+                );
+                status
+            })?;
+
+        _ = self.radio_threshold_report_sink.write(report, []).await;
+
+        Ok(Response::new(RadioThresholdReportRespV1 {
+            id: timestamp.to_string(),
+        }))
+    }
+
+    async fn submit_invalidated_threshold_report(
```

Comment:
Shouldn't this function also write out to the sink? Why silently drop?

Reply:
This is intentional, just a placeholder.  Another PR will fully support invalidating the radio for boosted rewards